MapboxNavigation/NavigationMapView.swift

```diff
@@ -1156,7 +1156,7 @@ public protocol NavigationMapViewDelegate: class {
      - parameter waypoints: The waypoints to be displayed on the map.
      - returns: Optionally, a `MGLShape` that defines the shape of the waypoint, or `nil` to use default behavior.
      */
-    @objc(navigationMapView:shapeDescribingWaypoints:)
```

Comment:
We should document these changes in the changelog.

Reply:
@bsudekum Sure. In this case where the function signature didn't necessarily change, what kind of changes should we document in the changelog?

Reply:
@bsudekum 👉  https://github.com/mapbox/mapbox-navigation-ios/pull/1338/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12